apps/fluent-tester/src/TestComponents/Badge/BadgeTest.tsx

```diff
@@ -9,19 +10,26 @@ import type { TestSection, PlatformStatus } from '../Test';
 import { Test } from '../Test';
 
 const badgeSections: TestSection[] = [
-  {
-    name: 'Basic Badge',
-    testID: BADGE_TESTPAGE,
-    component: BasicBadge,
-  },
+  Platform.select({
+    android: null,
+    default: {
+      name: 'Basic Badge',
+      testID: BADGE_TESTPAGE,
+      component: BasicBadge,
+    },
+  }),
   {
     name: 'Counter Badge',
+    testID: BADGE_TESTPAGE,
```

Comment:
Will it be an issue that two components have the same testID?

Reply:
No, these IDs are used to verify whether the test page has been loaded properly, as long as a section matches, the verification will pass successfully